test/README.md

```diff
@@ -175,6 +175,13 @@ On non-Windows platforms, this currently returns true.
 
 Platform normalizes the `dd` command
 
+### expectsError(code[, type[, message]])
```

Comment:
nit: I know the list is not completely in alphabetical order but can you move this below `enoughTestMem`?